### PR TITLE
Clock update

### PR DESCRIPTION
**Pull request by OmarGM2294**
What's included?

- Refactor for html
- Animations fixed
- Responsive fixed
- Figures moved to CSS from HTML

### CODE REVIEW

File: .firebaserc [hunk 1]
```diff
@@ -15,4 +15,4 @@
     "brianegan": "brianegan-flutter-staging",
     "xster-staging": "flutter-website-staging-6f2ad"
   }
-}
\ No newline at end of file
+}
```
Comment (1Iv4N) [inline .firebaserc]: Please revert this file.

File: src/_assets/image/clock/left_rectangle.svg [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (1Iv4N) [inline src/_assets/image/clock/left_rectangle.svg]: Please optimize the SVG

File: src/_assets/js/countdown.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
+document.addEventListener('DOMContentLoaded', () => {
```
Comment (1Iv4N) [inline src/_assets/js/countdown.js:1]: Please wrap this code in a IIFE.

File: src/_assets/js/countdown.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+            const hours = digitsHandler(remainingTime.hours);
+            const minutes = digitsHandler(remainingTime.minutes);
+            const seconds = digitsHandler(remainingTime.seconds);
+
+            daysContainer.innerText = `${days}`;
+            hoursContainer.innerText = `${hours}`;
+            minutesContainer.innerText = `${minutes}`;
+            secondsContainer.innerText = `${seconds}`;
+        } else {
+            console.log('The event is now');
```
Comment (1Iv4N) [inline src/_assets/js/countdown.js]: Please remove console.log.

File: src/_assets/js/countdown.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+            clearInterval(interval);
```
Comment (1Iv4N) [inline src/_assets/js/countdown.js:49]: Can we include one last print with "00" for all values before clearing the interval? just to make sure the "00" values appear if something goes wrong.

File: src/clock/index.html [hunk 6]
```diff
@@ -0,0 +1,33 @@
+---
+title: Clock
+layout: base
+show_banner: False
+show_footer: False
+body_class: clock
+---
+
+<section>
```
Comment (1Iv4N) [inline src/clock/index.html]: Let's try to clean the HTML.

**Review by 1Iv4N — CHANGES_REQUESTED** [review]

File: .firebaserc (hunk 1 above)
Comment (OmarGM2294) [reply .firebaserc]: Done

File: src/_assets/image/clock/left_rectangle.svg (hunk 2 above)
Comment (OmarGM2294) [reply src/_assets/image/clock/left_rectangle.svg]: Done

File: src/_assets/js/countdown.js (hunk 4 above)
Comment (OmarGM2294) [reply src/_assets/js/countdown.js]: Done

File: src/_assets/js/countdown.js (hunk 5 above)
Comment (OmarGM2294) [reply src/_assets/js/countdown.js:49]: Done

File: src/clock/index.html (hunk 6 above)
Comment (OmarGM2294) [reply src/clock/index.html]: Done

File: src/_assets/css/_clock.scss [hunk 7]
```diff
@@ -0,0 +1,214 @@
...
+}
+
+@keyframes geometric-small-background {
+  from {
+    background-position: bottom 90px left -144px, top right -140px, bottom -164px left -100px;
+  }
+  to {
+    background-position: bottom 90px left, top right, bottom left;
+  }
+}
```
Comment (1Iv4N) [inline src/_assets/css/_clock.scss]: Let's use a prefix for the keyframes.

File: src/_assets/css/_clock.scss [hunk 8]
```diff
@@ -0,0 +1,214 @@
...
+    width: 100%;
+    z-index: 99;
+  }
+
+  .heading {
+    animation: slide-in 1s .3s;
+    animation-fill-mode: both;
+    color: $site-color-white;
+    font-size: 18px;
+    margin: 0 0 24px 0;
```
Comment (1Iv4N) [inline src/_assets/css/_clock.scss]: You can omit the last cero. `margin: 0 0 24px;`

File: src/_assets/css/_clock.scss [hunk 9]
```diff
@@ -0,0 +1,214 @@
...
+  
+  background-color: $flutter-dark-blue-texture;
+  font-family: $site-font-family-alt;
+
+  .center {
+    position: absolute;
+    top: 53%;
+    transform: translateY(-50%);
+    width: 100%;
+    z-index: 99;
```
Comment (1Iv4N) [inline src/_assets/css/_clock.scss]: 2 or 3 should do the work.

File: src/_assets/css/_clock.scss [hunk 10]
```diff
@@ -0,0 +1,214 @@
...
+    }
+  }
+
+  .share-twitter__paragraph {
+    animation: slide-in 1s .6s;
+    animation-fill-mode: both;
+    color: $site-color-white;
+    font-size: 14px;
+    margin-bottom: 0px;
+    margin-top: 20px;
```
Comment (1Iv4N) [inline src/_assets/css/_clock.scss]: - Please use the shorthand whenever possible. 
- Please remove unit when the value is cero.

**Review by 1Iv4N — CHANGES_REQUESTED** [review]

File: src/_assets/css/_clock.scss (hunk 7 above)
Comment (OmarGM2294) [reply src/_assets/css/_clock.scss]: Done

File: src/_assets/css/_clock.scss (hunk 8 above)
Comment (OmarGM2294) [reply src/_assets/css/_clock.scss]: Done

File: src/_assets/css/_clock.scss (hunk 9 above)
Comment (OmarGM2294) [reply src/_assets/css/_clock.scss]: I changed this, It was weird, so I just use flex

File: src/_assets/css/_clock.scss (hunk 10 above)
Comment (OmarGM2294) [reply src/_assets/css/_clock.scss]: Done

**Review by 1Iv4N — APPROVED** [review]

**Pull request merged**